src/screens/Ticketing/Purchase/Overview/components/zone-item.tsx

```diff
@@ -71,6 +71,9 @@ export default function ZoneItem({
 }
 
 const useStyles = StyleSheet.createThemeHook((theme) => ({
+  container: {
+    marginVertical: theme.spacings.medium,
+  },
```

Comment:
Made a suggestion to how this should be handled for all components. See ae152883e3ccba2777fce4adb3f5b5cbda2b0f9c.

It's pretty clear from figma that every selection component should have 12px above and below, so I thought it would be clean to implement it the same way in the app.

<img width="476" alt="Screenshot 2022-05-05 at 11 09 47" src="https://user-images.githubusercontent.com/1774972/166894521-b1cfa005-94eb-42f5-acdb-f2ecc5492924.png">

Reply:
It's better this way when it is styled from outside the component :+1:

We'll keep it like this, even though It is kind of in conflict with the principle that styles and components should be a 1-to-1 relationship. The principle is there to ensure that styles are named based on in which component they are used (like "purchaseButton", "noTicketWarning", "headerTitle" etc.) and not have generic naming based on what they are doing (like "disabled", "bottomMargined", "strikeThrough" etc.). Even though this is not 1-to-1 relationship, it is not a totally generic naming.